tools/autograd/templates/python_functions.h

```diff
@@ -6,6 +6,18 @@
 
 namespace torch { namespace autograd { namespace generated {
 
-void initialize_autogenerated_functions();
+void initialize_autogenerated_functions_0();
```

Comment:
Should this be added via a template as well? Otherwise, we should add a comment in the python file where we specify the number of shards explaining that this will need to be updated manually if the number of shards is modified.

Reply:
Moved into `gen_autograd_functions.py`, and is now generated by a for-loop.